main.go

```diff
@@ -86,7 +86,12 @@ func (s *admissionWebhookServer) Review(ctx context.Context, in *admissionv1.Adm
 	annotation := podMetaPtr.Annotations[s.config.Annotation]
 
 	// use namespace annotation only if resource doesn't have it's own
```

Comment:
Minor comment:
We can move this comment below

Reply:
I agree, forgot to move when new if statement was added. Fixed 👍